client/src/components/Navbar/Navbar.js

```diff
@@ -7,6 +7,8 @@ import ResourcesDropdown from "./ResourcesDropdown";
 import AboutDropdown from "./AboutDropdown";
 import LoginIcon from "./LoginIcon";
 
+import SearchBox from "./SearchBox";
+
 function Navbar() {
   const [click, setClick] = useState(false);
   const [dropdown, setDropdown] = useState(false);
```

Comment:
dropdown1 and dropdown2 and not specific enough
please use more specific names for them
example: aboutMenueActive, resourcesMenueActive